hana_recipe.yaml

```diff
@@ -0,0 +1,15 @@
+source:
+  type: datahub_sap_hana.ingestion.HanaSource
+  config:
+    username: HOTEL
+    password: Localdev1
+    database: HXE
+    host_port: localhost:39041
+    schema_pattern:
+      deny: 
+        - .*SYS*
+    include_view_lineage: true
+sink:
+  type: file
+  config: 
+    filename: ./integration_tests/hana_golden_file.json
```

Comment:
I dont' think the base example in the root of the repo should sink to the golden file in the integration tests. If this is a test file, then it should live in test folder with the other integration test files. 

If it is just an example, which I think is a good idea, then it should use the console output.